src/Queue/Worker.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace CacheWerk\BrefLaravelBridge\Queue;
+
+use Illuminate\Contracts\Queue\Job;
+use Illuminate\Queue\WorkerOptions;
+use Illuminate\Queue\Worker as LaravelWorker;
+
+class Worker extends LaravelWorker
+{
+    /**
+     * Creates a new SQS queue handler instance.
+     *
+     * @param  \Illuminate\Contracts\Queue\Job  $job
+     * @param  string  $connectionName
+     * @param  \Illuminate\Queue\WorkerOptions  $options
+     * @return void
+     */
+    public function runSqsJob(Job $job, string $connectionName, WorkerOptions $options): void
+    {
+        $this->runJob($job, $connectionName, $options);
```

Comment:
Vapor uses it's own [`VaporWorker`](https://github.com/laravel/vapor-core/blob/2.0/src/Queue/VaporWorker.php) class which uses `pcntl` to handle timeouts set for jobs, should we copy that?

Reply:
added